### PR TITLE
Added mock data in sql.data and endpoint url's for pincode_ward ids for all services.

### PR DESCRIPTION
**Pull request by Basavaraj-Dundappa**

### CODE REVIEW

File: covid19zonecare/src/main/java/com/cognizant/callforcode/zonecare/entity/Pincode.java [hunk 1]
```diff
@@ -6,9 +6,10 @@
 @Entity
 public class Pincode {
 	
-	@Id
+	
 	private int Id;
 	private String pincode_ward;
+	@Id
```
Comment (ShruthiAjith) [inline covid19zonecare/src/main/java/com/cognizant/callforcode/zonecare/entity/Pincode.java:12]: is Pincode the primary key here? @DeepaBaddur

Comment (Basavaraj-Dundappa) [reply covid19zonecare/src/main/java/com/cognizant/callforcode/zonecare/entity/Pincode.java:12]: before id was primary it is autogenerated, which we can't track. So i made it pincode is primary key.

Comment (DeepaBaddur) [reply covid19zonecare/src/main/java/com/cognizant/callforcode/zonecare/entity/Pincode.java:12]: Pincode can be repeated here. Please check. There will be multiple wards for one pincode. Please make pincode_ward primary key or keep id as it is.

**Comment by Basavaraj-Dundappa** [comment]
i made primary key as pincodward

**Pull request closed without merging**